src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -149,6 +184,114 @@ private function renderEditForm(array $parameters = [])
+        $responseBuilder = $this->get('prestashop.bundle.grid.response_builder');
+
+        return $responseBuilder->buildSearchResponse(
+            $this->get('prestashop.core.grid.definition.factory.feature'),
+            $request,
+            FeatureGridDefinitionFactory::GRID_ID,
+            'admin_features_index'
+        );
+    }
+
+    /**
+     * @AdminSecurity("is_granted('delete', request.get('_legacy_controller'))", redirectRoute="admin_features_index")
+     *
+     * @param int $featureId
+     *
+     * @return RedirectResponse
+     */
+    public function deleteAction(int $featureId): RedirectResponse
+    {
+        try {
+            $this->getCommandBus()->handle(new DeleteFeatureCommand($featureId));
+            $this->addFlash(
+                'success',
+                $this->trans('Successful deletion.', 'Admin.Notifications.Success')
+            );
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->redirectToRoute('admin_features_index');
+    }
+
+    /**
+     * @AdminSecurity("is_granted('delete', request.get('_legacy_controller'))", redirectRoute="admin_features_index")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteAction(Request $request): RedirectResponse
+    {
+        $featureIds = $this->getBulkFeaturesFromRequest($request);
+
+        try {
+            $this->getCommandBus()->handle(new BulkDeleteFeatureCommand($featureIds));
+            $this->addFlash(
+                'success',
+                $this->trans('Successful deletion.', 'Admin.Notifications.Success')
+            );
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->redirectToRoute('admin_features_index');
+    }
+
+    /**
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
+     *     redirectRoute="admin_features_index"
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function updatePositionAction(Request $request)
+    {
+        $positionsData = [
+            'positions' => $request->request->get('positions'),
+        ];
+        $positionDefinition = $this->get('prestashop.core.grid.feature.position_definition');
+        $positionUpdateFactory = $this->get('prestashop.core.grid.position.position_update_factory');
+        try {
+            $positionUpdate = $positionUpdateFactory->buildPositionUpdate($positionsData, $positionDefinition);
+            $updater = $this->get('prestashop.core.grid.position.doctrine_grid_position_updater');
+            $updater->update($positionUpdate);
+            $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
```

Comment:
```suggestion
            $this->addFlash('success', $this->trans('Update successful', 'Admin.Notifications.Success'));
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/15045/commits/47cfcb666e63d342fe83809e6cd4f4dd7a326327